packages/@yoda/network/index.js

```diff
@@ -0,0 +1,316 @@
+'use strict'
+
+/**
+ * @module @yoda/network
+ * @description Provides classes to manage network functions on the device.
+ */
+
+var crypto = require('crypto')
+var EventEmitter = require('events').EventEmitter
+var FloraComp = require('@yoda/flora/comp')
+
+/**
+ * @class
+ * @auguments EventEmitter
+ * @param {boolean} subscribe - subscribe network.status or not
+ * @memberof module:@yoda/network
+ */
+class NetworkAgent extends EventEmitter {
+  constructor (subscribe) {
+    super()
+
+    this._callTarget = 'net_manager'
+    this._callCommand = 'network.command'
+    this._callTimeout = 60 * 1000
+
+    this._fid = '@network' + crypto.randomBytes(2).toString('hex')
+    this._flora = new FloraComp(this._fid)
```

Comment:
Do not use flora/comp outside of runtime. Use FloraAgent from `flora/index.js` directly.

Reply:
I used flora agent in the first edition, but someone tell me that I should use FloraComp.
I'm curious why this happened.

Reply:
I should call @yonzkon to use `FloraComp`, @legendecas could you explain why do not use comp here?

Reply:
FloraComp constructs with a name, which was used as the connection name to the flora dispatcher. With the name specified, other clients with same name would not be able to connect to flora dispatcher with such a *module*.

Reply:
Is `FloraComp` for server-side only?

Reply:
Yes, it's an syntactic sugar for registering lots of listeners and methods.

Reply:
Ok, sorry about that @yonzkon May I ask you to change to FloraAgent here :(
And BTW, if we need a pair of classes just like: `FloraClient` and `FloraService` which based on `FloraAgent`?

Reply:
@yorkie Yes, I'll try.
But I have no passion in it now, as it's too difficult to submit a patch which meets the standards.